api/proto/info/info.proto

```diff
@@ -243,6 +243,27 @@ message ZInfoAppInstance {
   string domainName = 3;
 }
 
+enum ZSimcardState {
+  SIM_INVALID     = 0;
+  SIM_ASSIGNED    = 1;
+  SIM_PROVISIONED = 2;
+  SIM_ACTIVE      = 3;
+  SIM_SUSPENDED   = 4;
+  SIM_CANCELLED   = 5;
+}
+
+message ZSimcardInfo {
+    string          imsi = 1;
+    string          iccid = 2;
+    ZSimcardState   state = 3;
+}
+
+message ZCellularModuleInfo {
+    string imei = 1;
+    string firmwareVersion = 2;
```

Comment:
I've started using firmware_version for new fields. The go struct name is unchanged i.e., the CamelCase FirmwareVersion as what you get from this. So just a local change in the proto files to do that for your new fields and keep yetus happy.

Reply:
Hmm.. Interesting.. Made the change.